pyracf/common/security_admin.py

```diff
@@ -55,6 +61,7 @@ def __init__(
         generate_requests_only: bool = False,
         add_field_data: Union[dict, None] = None,
         overwrite_field_data: Union[dict, None] = None,
+        add_more_secrets: Union[dict, None] = None,
```

Comment:
I would call this field `additional_secret_traits`.

Reply:
per above, agree.